src/xdocs/config_imports.xml

```diff
@@ -818,6 +822,42 @@ import android.*;
           which is part of the Checkstyle distribution.
         </p>
 
+        <h4 id="blacklist-example">Example of blacklist mode</h4>
+        <p>
+          Short description of the behaviour:
+          <ul>
+            <li>Check starts checking from the closest to the current package described inside import control file.</li>
+            <li>If there is matching allow/disallow rule inside the current package then check returns "allowed" or "disallowed" message.</li>
+            <li>If there is no matching allow/disallow rule inside the current package then checks goes up to the parent and continues checking.</li>
+            <li>If there is no matching allow/disallow rule inside the root level then import is disallowed by default.</li>
+          </ul>
+          To have a <b>blacklist mode</b>, it is required to have disallows inside subpackage and allow bigger package inside parent package.
+          In the example below inside subpackage <code>com.puppycrawl.tools.checkstyle.filters</code> any import from <code>java.util</code> package except <code>java.util.Map</code> is allowed, for example <code>java.util.List</code> or <code>java.util.Date</code>.
+        </p>
+        <source>
+&lt;import-control pkg=&quot;com.puppycrawl.tools.checkstyle&quot;&gt;
+    &lt;allow pkg=&quot;java.util&quot;/&gt;
+    &lt;subpackage name=&quot;filters&quot; &gt;
+        &lt;disallow class=&quot;java.util.Map&quot;/&gt;
+    &lt;/subpackage&gt;
+&lt;/import-control&gt;
+        </source>
+        <p>
+          In the next example imports <code>java.util.stream.Stream</code> and <code>java.util.stream.Collectors</code> are disallowed inside <code>com.puppycrawl.tools.checkstyle.checks.imports</code> package,
+          but because of <code>&lt;allow pkg=&quot;java.util&quot;/&gt;</code> every import is allowed except described ones.
```

Comment:
`java.util` is not the package of `java.util.stream.Stream`. `java.util.stream` is.
I am not sure if this was the purpose of this example or not.

"every import is allowed " seems like too broad a statement to make.
Only all `java.util` package imports are allowed.